modules/channel-web/src/config.ts

```diff
@@ -27,4 +27,8 @@ export interface Config {
    * @default 6 hours
    */
   recentConversationLifetime?: string
+  /**
+   * @default 360
+   */
+  messageLength?: number
```

Comment:
Like Yann suggested, I'd prefer to name if `maxMessageLength`

Reply:
@slvnperron fixed